static/app/views/monitors/monitorForm.tsx

```diff
@@ -48,7 +48,7 @@ class MonitorForm extends Component<Props> {
   form = new MonitorModel();
 
   formDataFromConfig(type: MonitorTypes, config: MonitorConfig) {
-    const rv = {};
+    const rv: Record<string, ScheduleType | number | unknown> = {};
```

Comment:
does `unknown` just make the whole thing `unknown`?

Reply:
Yeah, it allows for anything to write to it... I just added it because iirc config.schedule_type can have a unknown type 😞